cluster/ci/config/prow/config.yaml

```diff
@@ -3209,6 +3209,33 @@ presubmits:
         - --template=/usr/local/e2e-aws
         - --target=e2e-aws
 
+  - name: shell-check
+    trigger: "(?m)^/shell-check"
+    rerun_command: "/shell-check"
```

Comment:
As in the other PR, convention is to use `/test <name>` and have context `ci/prow/<name>`